kratos/containers/container_expression/expressions/literal/literal_flat_expression.h

```diff
@@ -89,13 +87,15 @@ class KRATOS_API(KRATOS_CORE) LiteralFlatExpression : public Expression {
 
     const std::vector<IndexType> GetShape() const override;
 
-    inline IndexType DataSize() const noexcept { return mData.DataSize(); }
+    IndexType DataSize() const noexcept { return mData.DataSize(); }
+
+    TRawDataType* begin() noexcept { return mData.begin(); }
 
-    inline TRawDataType* DataBegin() noexcept { return mData.DataBegin(); }
+    TRawDataType* end() noexcept { return mData.end(); }
 
-    inline TRawDataType const* DataBegin() const noexcept { return mData.DataBegin(); }
+    TRawDataType const* cbegin() const noexcept { return mData.cbegin(); }
 
-    inline TRawDataType const* DataEnd() const noexcept { return mData.DataEnd(); }
+    TRawDataType const* cend() const noexcept { return mData.cend(); }
```

Comment:
```suggestion
    TRawDataType* begin() noexcept { return mData.begin(); }

    TRawDataType* end() noexcept { return mData.end(); }

    const TRawDataType* begin() const noexcept { return mData.begin(); }

    const TRawDataType* end() const noexcept { return mData.end(); }

    TRawDataType const* cbegin() const noexcept { return mData.cbegin(); }

    TRawDataType const* cend() const noexcept { return mData.cend(); }
```

Reply:
(I just added `const` versions of `begin` and `end`)

Reply:
done